frontend/src/components/Navbar/Navbar.jsx

```diff
@@ -1,11 +1,42 @@
 import "./Navbar.css";
+import axios from "axios";
+
+export default function Navbar({ user }) {
+    async function handleLogout() {
+        try {
+            await axios.post("http://localhost:3000/users/logout", {
+                withCredentials: true
+            });
+            localStorage.clear();
+        } catch (error) {
+            console.log(error);
+        }
+    }
+
+    function handleLogin() {
+        window.location.href = "/login";
+    }
 
-export default function Navbar() {
     return (
         <nav className="navbar">
-            <a href="/home">Stock Zone</a>
-            <a href="/home">Home</a>
-            <a href="/search">Search</a>
+            <div className="nav-pages">
+                <a href="/home">Stock Zone</a>
+                <a href="/home">Home</a>
+                <a href="/search">Search</a>
+            </div>
+            <div className="nav-profile">
+                {!user && (
```

Comment:
I think both the user and !user cases can be handled with one ternary `user ? ` instead of two AND operations.